pytorch_blade/src/pybind.cpp

```diff
@@ -49,7 +49,6 @@ std::string pybind_version() {
 
 template <>
 void initModules<false>(py::module& m) {
-  torch::blade::backends::InitTorchBladeEngine();
```

Comment:
How about `InitBladeDiscEngine`? There seems no global initializer for it.

Reply:
This will be used in TorchDISC & TorchBlade

Reply:
Does a C++ user of TorchBlade still have to call `InitBladeDiscEngine` before `torch::jit::load` a optimized model ?

Reply:
Nop